index.js

```diff
@@ -25,12 +30,19 @@ function destroyFnMap(service) {
     });
 }
 
+function addSelfExclude(pkg) {
```

Comment:
This might be easier to follow as:

```js
const EXCLUDE = ['node_modules/serverless-cljs-plugin']
```

And in the constructor do:

```js
_.update(
  serverless.service, 'package.exclude', v => _.concat(v || [], EXCLUDE))
```

Reply:
wow I thought this was like Clojure's `update`, but it is a setter instead :smile: